### PR TITLE
Adding best practices tutorial

### PR DESCRIPTION
**Pull request by ethangj**
Feature flag best practices blog, slightly updated and porting to HDH.

### CODE REVIEW

**Comment by ravilach** [comment]
@KimberlyFields @ethangj this might not be a tutorial more of a best practices. Better place for this is in the docs?

**Comment by KimberlyFields** [comment]
@StrawberryPad please work with Ethan on finding the best place for this content.

**Comment by ethangj** [comment]
@KimberlyFields @ravilach Given that we just lack content right now, would it be possible to get it up there so to pad things out and as we get more content over q3 we can move it later?

**Comment by ravilach** [comment]
IMO it's not a tutorial but a great best practices guide. @StrawberryPad any guidance to put in FF docs? CC @KimberlyFields

File: tutorials/feature-flags/best-practices.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+Another feature flag management oversight that many teams make is not having clear internal process or agreement around who owns removing stale flags. Or how often they want that process to happen. This can often lead to teams feeling there are too many feature flags or that they’re stuck in feature flag hell. That’s not where you want to be if you want to reap the benefits and continue to create more feature flags, especially in a production environment.
+
+One of the benefits of flagging is allowing teams all across the company to use flags for a wide variety of different scenarios such as experiment toggles to test user behavior, ops toggles, using backend flags to modify system behavior. 
+
+The downside is, with many new features and anyone having access to implement feature flags, you will find that your application code can have feature flags all over the place. In addition to having feature flags spread across the code base, the feature flag management system will also grow as each one of these new feature flags is established in the code. In other words, as use of feature flags grows within the organization, there are two places where hygiene needs to be maintained.
```
Comment (StrawberryPad) [inline tutorials/feature-flags/best-practices.md:66]:
```suggestion
The downside is, with many new features and anyone having access to implement feature flags, you will find that your application code can have feature flags all over the place. In addition to having feature flags spread across the code base, the feature flag management system will also grow as each one of these new feature flags is established in the code. In other words, as use of feature flags grows within the organization, these are two places where hygiene needs to be maintained.
```

File: tutorials/feature-flags/best-practices.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+![Example Pipeline With Governance Controls](static/ff-best-practices/example-pipeline.png)
+
+## Conclusion
+
+Feature flagging can and will speed up your team, reduce the volume and severity of incidents, and make it easier than ever before to be reactive to your customers and to be a learning-focused organization. 
+
+By implementing the right tools, practices, and processes at your organization, you’ll find that you’re able to go the full mile with feature flags and really ensure that you’re getting the most value out of them. Hopefully, with these feature flags best practices you will see smooth sailing and find more ways to use flags.
+
+If you’re looking for a way to implement these best practices and ensure that you have a solution that supports your ability to do so, and maybe even have some baked in, it’s well worth your time to check out Harness Feature Flags. When you’re ready, you’re welcome to sign up for a demo or a free trial! Happy coding.
```
Comment (StrawberryPad) [inline tutorials/feature-flags/best-practices.md:146]:
```suggestion
If you’re looking for a way to implement these best practices and ensure that you have a solution that supports your ability to do so, and even have some best practices baked in, it’s well worth your time to check out Harness Feature Flags. When you’re ready, you’re welcome to sign up for a demo or a free trial! Happy coding.
```

**Review by StrawberryPad — CHANGES_REQUESTED** [review]

**Comment by StrawberryPad** [comment]
@ethangj - added suggestions mostly to comply with our heading style, but also some for clarification. I'm happy to put this within tutorials to pad the content. It doesn't really fit into the docs as written, since it's more like a blog. We do have a best practices section in docs, which is appropriately short. I can point from that section to this article/"tutorial" for more info, once we merge this.

**Pull request merged**